app/models/manageiq/providers/openshift/container_manager.rb

```diff
@@ -8,6 +8,8 @@ class ManageIQ::Providers::Openshift::ContainerManager < ManageIQ::Providers::Co
   require_nested :RefreshWorker
   require_nested :Refresher
 
+  delegate :delete_project, :to => :connect
```

Comment:
AFAIK we don't have the appropriate role for the service account to delete projects in the cluster, do we add it anywhere?
cc @simon3z